umiCollapse.wdl

```diff
@@ -139,10 +157,18 @@ workflow umiCollapse {
             Bams = bwaMem.bwaMemBam
     }
 
+    if (doBamQC) {
+        call bamQC.bamQC as preDedupBamQC {
+            input:
+                bamFile = mergeLibrary.mergedBam ,
+                outputFileNamePrefix = "~{outputPrefix}.preDedup"
+        }
+    }
+
     scatter (umiLength in umiLengths) {
         call bamSplitDeduplication {
             input:
-                bamFile = mergeLibrary.mergedBam,
+                bamFile = mergeLibrary.mergedBam ,
```

Comment:
that extra space may not be harmful but looks a bit annoying

Reply:
Done all your suggestions, thanks!